src/oidcprovider/__tests__/router.test.ts

```diff
@@ -50,7 +50,11 @@ describe("interactionLogic", () => {
     const detail = records.loginPromptDetail;
     const mockUserInfoClient = phonies.makeMockUserInfoClient();
     mockUserInfoClient.findUserByFederationToken.mockReturnValue(
-      TE.right({ id: cookieValue })
+      TE.right({
+        fiscalCode: cookieValue,
```

Comment:
Can we move this information to `records.ts`?

Reply:
Done!